### ISSUE TITLE
Cannot get `LINE history` if code of the line contains Chinese.

### ISSUE THREAD

**Issue by Modnars** [Axosoft/vscode-gitlens#1713]
- GitLens Version: v11.6.1
- Git Version: git version 2.19.1

- VSCode Version: 1.62.0, x64
- OS Version: Linux version 3.10.107-1-tlinux2_kvm_guest-0050 (gcc version 4.8.2 20140120 (Red Hat 4.8.2-16) (GCC)

Steps to Reproduce:
Look at the screenshot:
1: I can see the line history in English normally.
![image](https://user-images.githubusercontent.com/22772190/140718783-400e44eb-ded2-4a77-b94a-8b2559d79183.png)

2: I can not see the line history in Chinese?
![image](https://user-images.githubusercontent.com/22772190/140720609-855fd288-1f10-4085-a89c-078a3b85be94.png)

3. I can not see the line history if the code contains Chinese?
![image](https://user-images.githubusercontent.com/22772190/140720743-70b07090-554a-4895-8bcf-2153f2504dd3.png)

Is this a custom function? If so, could you tell me how to switch it? Thanks.

**Comment by eamodio** [comment]
Opened https://github.com/microsoft/vscode/issues/136690 to track this issue.

**Comment by Modnars** [comment]
Oh... my VS-Code client's `OS Version` is:
```bash
ProductName:	macOS
ProductVersion:	11.3.1
BuildVersion:	20E241
```
The origin issue's `OS Version` is my remote develop environment's version, thanks:)

**Comment by eamodio** [comment]
This was fixed by VS Code 1.62.3.
